README.md

```diff
@@ -1,24 +1,180 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <h3><b>Hello Rails React</b></h3>
 
```

Comment:
- [ ] In your `db/seed.rb` file, you should add code that will seed the initial greeting on the database. Like so
Also, include the instruction on how to seed the data in your README getting started section. 

```ruby
5.times do | index |
  Greeting.create(message: "Greetings, #{ index}")
end
```